### ISSUE TITLE
Use Relavant Path for Files Location

### ISSUE THREAD

**Issue by Riajyuu** [NickeManarin/ScreenToGif#478]
As per #6, config file is now use relevant path.
But extra components (ffmpeg, gifski) & project folder are still absolute path.
Is there any specific reason for this?

**Comment by NickeManarin** [comment]
You can use relative paths with those settings. Just type a period (.) and the fields will be treated as relative.

**Comment by vatterspun** [comment]
Related #295 and #6

**Comment by Riajyuu** [comment]
Sorry for my delay on this:
Yes, you can change the path by adding a period after downloaded. But I mean it shall be displayed as set as relative expression by default.

**Comment by stephen147** [comment]
It would be nice once you choose to install ffmpeg and Gifsi that it will re-route to the relative path.

And the settings file appears to be absolute also out of the box.

![image](https://user-images.githubusercontent.com/14054664/61182402-4a7c2d00-a62a-11e9-9dd0-9d0ef4193189.png)

**Comment by NickeManarin** [comment]
@Riajyuu Maybe if the user saves Gifski/FFmpeg in a path that is inside the same path as the App, the path could be set as relative by default. If the user saves it to elsewhere, it will be displayed the full path.

**Comment by NickeManarin** [comment]
@stephen147 Yes, the settings file is absolute when inside the AppData, but it is relative when inside the local folder.
